tar/test/test_stdio.c

```diff
@@ -22,6 +22,12 @@ static const char *xvf_err =
 "x f\n"
 "x l\n";
 
+/*
+ * This string should appear in the verbose listing regardless of platform,
+ * locale, username, or groupname.
+ */
+const char * tvf_contains = "l link to f";
```

Comment:
Whoops, that should have been `static const char *`.  This now fails `clang -Wmissing-variable-declarations`.  I "checked" that it compiled by running `make all`, but forgot that I needed `make check` to compile the tests.

I have a fix in my "misc fixes" branch, which I'll upload once some of the bigger diffs are out of the way.